src/plugins/LADTable/components/LadRow.vue

```diff
@@ -237,13 +237,11 @@ export default {
     }
 
     this.previewAction = new PreviewAction(this.openmct);
-    this.previewAction.on('isVisible', this.togglePreviewState);
```

Comment:
`this.togglePreviewState` was undefined, and this wasn't doing anything except making our unit tests fail.

Reply:
I think you're right. I wonder if our preview action changed at some point as it doesn't seem to have much benefit to emitting this 'isVisible' event at all.

Reply:
> It seems that the new version throws an exception if you try to unregister a listener that doesn't exist, or register a listener without context.

From v3.0.0 onwards, EventEmitter3 [throws an error](https://github.com/primus/eventemitter3/blob/5.0.1/index.js#L60) if you try to register a listener with a callback that is not a function

Reply:
Actually, this is being used for search results here: 
https://github.com/nasa/openmct/blob/master/src/ui/layout/search/SearchResultsDropDown.vue#L48
It was likely a copy-paste error where the `this.togglePreviewState` method was not added. It is also missing in `AnnotationSearchResult.vue`

You can see an implementation here: 
https://github.com/nasa/openmct/blob/master/src/ui/layout/RecentObjectsListItem.vue#L116